containers/daap-push-to-catalogue/src/var/task/push_to_catalogue.py

```diff
@@ -0,0 +1,104 @@
+import json
+import structlog
+import logging
+import os
+
+import boto3
+from data_platform_catalogue import CatalogueClient, CatalogueError
+from data_platform_catalogue import DataProductMetadata as omdProductMetadata
+from data_platform_catalogue import TableMetadata
+
+
+def handler(event, context):
+    # can't use daap-python-base as it's python 3.11 and need 3.10 for
+    # data_platform_catalogue, hence we can't use DataPlatformLogger here
+    structlog.configure(
+    wrapper_class=structlog.make_filtering_bound_logger(logging.INFO),
```

Comment:
```suggestion
    wrapper_class=structlog.make_filtering_bound_logger(logging.INFO),
```

this is the default so think you can skip this is as well